pokedex/src/components/display-pokemon.tsx

```diff
@@ -50,23 +51,16 @@ const PText = styled.p`
   border-radius: 6px;
 `
 
-interface Pokemon {
-  id: number;
-  name: string;
-  sprites: any;
-  height: number;
-  weight: number;
-  types: any;
+interface DisplayPokemonProps {
+    pokemonInfo?: Pokemon | Pokemon[]
 }
-
-
-function DisplayPokemon (props: any) {
+function DisplayPokemon ({pokemonInfo }: DisplayPokemonProps) {
   let pokemonList;
-  if (props.pokemonInfo) {
-    if (!props.isDataList) {
-      pokemonList = <ListItem>{infoCard(props.pokemonInfo)}</ListItem>
+  if (pokemonInfo) {
+    if (!Array.isArray(pokemonInfo)) {
```

Comment:
this check replaces "isList" (the variable itself is enough to determine if is list or not)